source/lac/vector.cc

```diff
@@ -27,6 +27,9 @@ namespace internal
   {
     template void copy_vector<int,double> (const dealii::Vector<int> &,
                                            dealii::Vector<double> &);
+
+    template void copy_vector<int,int> (const dealii::Vector<int> &,
+                                        dealii::Vector<int> &);
```

Comment:
exactly, as it was missing and without it tests cases with integeres will not compile.